shared/chat/selectable-big-team-channel-container.js

```diff
@@ -23,15 +22,14 @@ const mapStateToProps = (state, {conversationIDKey}) => {
 const mapDispatchToProps = () => ({})
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  const isSelected = ownProps.isSelected && !Styles.isMobile
   return {
     channelname: stateProps.channelname,
-    isSelected,
+    isSelected: ownProps.isSelected,
```

Comment:
@mmaxim I'm changing these back in this PR for now. Tried to figure out how it's related to the chat search but mobile chat search seems broken on master.